lmdeploy/pytorch/kernels/default/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
what's the package "default" used for?

Reply:
This is designed to place pure pytorch implementation, without any device assumption.
So if the backend does not provide a custom kernel, we can still perform the inference.